src/app/routes/cpsAsset/getInitialData/index.js

```diff
@@ -61,9 +61,9 @@ const processOptimoBlocks = toggles => service =>
     addBylineBlock,
     addRecommendationsBlock,
     addMpuBlock,
-    applyBlockPositioning,
-    // addExperimentPlaceholderBlocks(service),
+    addExperimentPlaceholderBlocks(service),
     addIdsToBlocks,
+    applyBlockPositioning,
```

Comment:
I'd possibly keep this commented out for now until all the variations are merged in.

My strategy was to get everything merged in, then remove the `addRecommendationsBlock` function a few lines above and instead use the `addExperimentPlaceholderBlocks` function. This will replicate the behaviour of `addRecommendationsBlock` for all services other than Hindi, but also include the `experimentBlocks` for Hindi.

Reply:
Cool, will comment it out for the time being.

Reply:
Done 👍